web/src/components/molecules/TopPage/index.tsx

```diff
@@ -33,10 +34,12 @@ const TopPage: React.FC<Props> = ({
   onPageChange,
   loadingPlugins,
   pageSize,
+  setCurrentVersion,
 }) => {
   return (
     <Wrapper>
-      {showBanner && <TitleBar />}
+      {/* NOTE: No banner in the new design */}
+      {/* {showBanner && <TitleBar />} */}
```

Comment:
If there is no more banner, remove this code and the TitleBar component completely.

Reply:
Thanks Kyle. This has been removed.